q2_diversity/tests/test_ordination.py

```diff
@@ -22,5 +23,40 @@ def test_pcoa(self):
                                   ids=['S1', 'S2', 'S3'])
         actual = pcoa(dm)
         expected = skbio.stats.ordination.pcoa(dm)
-        skbio.util.assert_ordination_results_equal(
+        assert_ordination_results_equal(
             actual, expected, ignore_directionality=True)
+
+    def test_pcoa_fsvd(self):
+        dm1 = skbio.DistanceMatrix([[0.0000000, 0.3333333, 0.6666667],
+                                   [0.3333333, 0.0000000, 0.4285714],
+                                   [0.6666667, 0.4285714, 0.0000000]],
+                                  ids=['S1', 'S2', 'S3'])
+        dm2 = skbio.DistanceMatrix([[0.0000000, 0.3333333, 0.6666667],
+                                    [0.3333333, 0.0000000, 0.4285714],
+                                    [0.6666667, 0.4285714, 0.0000000]],
+                                   ids=['S1', 'S2', 'S3'])
+
+        # Test eigh vs. fsvd pcoa and inplace parameter
+        expected_results = pcoa(dm1, method='fsvd', number_of_dimensions=3,
+                                inplace=False)
+
+        results = pcoa(dm2, method='fsvd', number_of_dimensions=3,
+                       inplace=False)
+
+        results_inplace = pcoa(dm2, method='fsvd', number_of_dimensions=3,
+                               inplace=True)
+
+        assert_ordination_results_equal(results, expected_results,
+                                        ignore_directionality=True,
+                                        ignore_method_names=True)
+
+        assert_ordination_results_equal(results, results_inplace,
+                                        ignore_directionality=True,
+                                        ignore_method_names=True)
+
+        # Test invalid parameters for number_of_dimensions
+        with self.assertRaises(ValueError):
+            pcoa(dm1, method='fsvd', number_of_dimensions=-1)
+
+        with self.assertRaises(ValueError):
+            pcoa(dm1, method='fsvd', number_of_dimensions=-100)
```

Comment:
These tests aren't necessary, the range predicate precludes number_of_dimensions being negative.